index.html

```diff
@@ -292,7 +293,7 @@ <h3 id="ruby-character-size-and-character-placement">Ruby character size and cha
 <li id="l20200529016">In vertical text, ruby annotation is placed to the right of the base characters,
     and the character frame of the ruby annotation is placed flush
     against the character frame of the base characters.
-	
+	(See also JLReq <a href="https://www.w3.org/TR/jlreq/#kanji_hiragana_and_katakana">section 2.1.2 Kanji, Hiragana and Katakana</a> for detail on character frame.)
```

Comment:
What about adding a link to the definition of "character frame" in jlreq directly in the previous sentence?

Reply:
https://www.w3.org/TR/jlreq/#term.character-frame targets one in the glossary table without list of where it's used, which just says:

> Rectangular area occupied by a character when it is set solid.

and I think it is helpful if we point to main text.

Reply:
note, this was fixed in d54f8eb